src/GitHub.App/Services/ModelService.cs

```diff
@@ -97,7 +97,7 @@ IObservable<IEnumerable<string>> GetOrderedGitIgnoreTemplatesFromApi()
         IObservable<IEnumerable<AccountCacheItem>> GetUser()
         {
             return hostCache.GetAndRefreshObject("user",
-                () => apiClient.GetUser().Select(AccountCacheItem.Create), TimeSpan.FromMinutes(5), TimeSpan.FromDays(7))
+                () => apiClient.GetUser().Select(x => new AccountCacheItem(x.User)), TimeSpan.FromMinutes(5), TimeSpan.FromDays(7))
```

Comment:
Just to keep with the convention of using `Create` methods on these calls, add an overload to `AccountCacheItem.Create` that takes a `UserAndScopes` object instead.